tutorials/heritage/htmlex.C

```diff
@@ -1,10 +1,11 @@
 /// \file
-/// \ingroup Tutorials
+/// \ingroup tutorial_heritage
 /// This file demonstrates how THtml can document sources.
 ///
 /// See the [Users Guide](https://root.cern.ch/root/htmldoc/guides/users-guide/ROOTUsersGuide.html)
-/// chapter [Automatic HTML Documentation](https://root.cern.ch/root/htmldoc/guides/users-guide/ROOTUsersGuideChapters/HTMLDoc.pdf),
-/// and [THtml's class documentation](https://root.cern.ch/doc/master/classTHtml.html).
+/// chapter [Automatic HTML
```

Comment:
Does splitting the line retain the link "Automatic HTML Documentation"?

Reply:
clang format kicked in, will fix.